README.md

```diff
@@ -51,3 +52,11 @@ services:
     environment:
       - TEST_OIDC_ALGO=hs
 ```
+
+Creating users
+--------------
+
+The `testprovider` image has a Django management command for creating users in
+the OIDC provider. This lets you create users on the command line.
+
+Usage: `/code/manage.py createuser USERNAME PASSWORD EMAIL`
```

Comment:
Maybe just mentioning something like the following would be enough:

```
Usage:
# With an already running testprovider container run:

> docker exec <container_name> /code/manage.py createuser USERNAME PASSWORD EMAIL
```

Reply:
Also Dockerfile defines:
```
WORKDIR /code
```

So this should also work

```
> docker exec <container_name> manage.py createuser USERNAME PASSWORD EMAIL
```

Reply:
I like that a lot better!